rpc/src/transaction_status_service.rs

```diff
@@ -306,6 +309,7 @@ pub(crate) mod tests {
 
     struct TestNotification {
         _meta: TransactionStatusMeta,
+        post_accounts_states: Vec<(Pubkey, AccountSharedData)>,
         transaction: SanitizedTransaction,
     }
```

Comment:
I'm going to be annoying here.

It is more consistent to use the same order as in the order locations we reference both `transactions` and `post_accounts_states`:

```suggestion
    struct TestNotification {
        _meta: TransactionStatusMeta,
        transaction: SanitizedTransaction,
        post_accounts_states: Vec<(Pubkey, AccountSharedData)>,
    }
```